WcaOnRails/db/migrate/20180206211650_add_locations_table.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AddLocationsTable < ActiveRecord::Migration[5.1]
+  def up
+    create_table :locations do |t|
+      t.integer :user_id, null: false
+      t.integer :latitude
+      t.integer :longitude
```

Comment:
In retrospect, I regret that we named these latitude and longitude. I think we're actually storing these in microdegrees, so how about `latitude_microdegrees` and `longitude_microdegrees` instead?